modules/installation-guide/partials/proc_importing-tls-certificates-to-che-server-java-truststore.adoc

```diff
@@ -1,92 +1,174 @@
 
 
 [id="importing-tls-certificates-to-{prod-id-short}-server-java-trustore_{context}"]
```

Comment:
Can you also update the `id` and all `xref` statements pointing to this `id`?

Reply:
Done